fstest/test_all/config.yaml

```diff
@@ -264,3 +264,9 @@ backends:
    remote:   "TestMailru:"
    subdir:   false
    fastlist: false
+ - backend:  "seafile"
+   remote:   "TestSeafile:"
+   fastlist: true
+   ignore:
+     - TestVFSStatfs
```

Comment:
I think this test was broken and now I've fixed it so you can probably remove this!